### DISCUSSION TITLE
Crash when setting ClearButtonVisibility property when styling Entry target type

### DISCUSSION THREAD

**Discussion by andu04** [dotnet/maui#11221] [Q&A]
Setting ClearButtonVisibility property in an Entry style will cause app crash:
```
 <Entry Text="Entry with clear">
    <Entry.Style>
        <Style TargetType="Entry">
            <Setter Property="ClearButtonVisibility" Value="WhileEditing"/>
        </Style>
    </Entry.Style>
</Entry>
```

However, if you set the property directly, it works without problems:
`
<Entry Text="Entry with clear" ClearButtonVisibility = "WhileEditing"/>
`

Does anyone know what I'm doing wrong? Has this happened to anyone?

**Comment by stevegdavis** [comment]
I have tried it as your 2 snippets, on the 1st snippet it doesn't crash but the Clear button doesn't appear, however as you stated your 2nd snippet works fine, sorry not really any help.

**Reply by andu04** [reply]
Sorry I forgot to mention that the crash occurs in .Net 7. In .Net 6 it's working also for me.

**Reply by stevegdavis** [reply]
I forgot to mention I am using .NET 7 to test your snippets - no crash. If you want someone to investigate you may need to supply a reproducible sample.
